tests/src/e2e/ledger_tests.rs

```diff
@@ -126,7 +126,7 @@ fn test_node_connectivity_and_consensus() -> Result<()> {
     // 4. Check that all the nodes processed the tx with the same result
     let mut validator_0 = bg_validator_0.foreground();
     let mut validator_1 = bg_validator_1.foreground();
-    let expected_result = "all VPs accepted transaction";
+    let expected_result = "successful txs: 1";
```

Comment:
I think there's an issue with this (and we actually already have it in some tests) that is, 1 tx will appear twice in "successful txs", once for the wrapper and then inner tx. Some of the existing tests that are waiting for "Transaction is valid" have the same issue

Reply:
I think ideally the wrapper tx result should use a different type and format differently in the client (related to #20) and maybe we should distinguish these in the ledger stats log too

Reply:
yeah, I agree, ill try to open a new PR to improve e2e